build.gradle

```diff
@@ -204,6 +204,14 @@ dependencyManagement {
   }
 }
 
+test {
+  useJUnitPlatform {
+  }
+  testLogging {
+    events("passed", "skipped", "failed")
```

Comment:
Would that also be interesting for the client?

Reply:
Probably.